kubernetes/windows/hostlogs/main.ps1

```diff
@@ -86,13 +86,19 @@ Set-EnvironmentVariables
 Start-FileSystemWatcher
 
 if(Get-GenevaEnabled){
-    Invoke-Expression ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe -useenv"
+    Write-Host "Starting Windows AMA in 1P Mode"
+
+    Start-Job -Name "WindowsHostLogsJob" -ScriptBlock { 
+        Start-Process -NoNewWindow -FilePath ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe" -ArgumentList @("-useenv")
+    }
+
+    [System.Environment]::SetEnvironmentVariable("HOSTLOGS_MA_STARTED", "true", "process")
+
 } else {
     Write-Host "Geneva not configured. Watching for config map"
```

Comment:
How is this watching for the config map? Or is this referencing something happening elsewhere?

Reply:
filesystemwatcher.ps1 is doing that: https://github.com/microsoft/Docker-Provider/blob/sky-dev/build/hostlogswindows/installer/scripts/filesystemwatcher.ps1